### DISCUSSION TITLE
Osu mania 4k argon skin discussion

### DISCUSSION THREAD

**Discussion by Alsen99** [ppy/osu#20622] [Q&A]
Hi, I've been playing with the osu mania 4k with the new argon skin for a while, Its a great skin with some getting used to, but I think there are a couple of things I want to suggest and want some opinion on to improve this skin, especially 4k (since I don't really play other keys but feel free to start for other keys discussion here too)

1. The Long Notes are in my opinion inconsistent

The new skin highlights the "bottom" of the notes, the judgement line is also highlighted, but why the tail note is highlighted at the "top"? I sometimes misread it to be where i supposed to realease but in reality its not.
For the long notes my suggestion is to use the triangles long notes where the notes are really thin making the "top" and "bottom" the same which make it easier to read.

2. For 4k the notes width is kinda thin
    Maybe this one is just preference and probably be in customization section instead but the notes are not wide enough for me.

Here are some pictures to illustrate my suggestions
![1](https://user-images.githubusercontent.com/54655737/194707024-7138d369-2261-47f9-8faa-f453079dc6fc.png)
![1a](https://user-images.githubusercontent.com/54655737/194707029-90d3f453-ef93-49b5-9530-d62c37859c33.png)

**Comment by Walavouchey** [comment]
It seems like the skin tries to have both a *receptor* (DDR-style) and a judgement *line* at the same time. The notes, hold note heads and hold note tails have the same height as the "judgement bar", suggesting that you should press/release when the note/head/tail overlaps the bar, while the highlighted lines suggest that you should match bright lines with  bright lines.

That may have not be an issue if the skin was designed with that in mind, just thought I'd elaborate.

Also this issue has been mentioned in https://github.com/ppy/osu/discussions/20616

**Reply by Alsen99** [reply]
Oh, I guess when I was drafting, #20616 posted first, whoops

**Comment by peppy** [comment]
Can agree with the white line being moved to be in line with the target area. It will be one of the areas we tweak going forward.

Column width is subjective but maybe we can expand lower key counts by default.
